doc/lsst.afw.multiband/multiband.rst

```diff
@@ -480,17 +480,14 @@ to load an exposure from a file:
 .. code-block:: python
 
     from lsst.afw.image import MultibandExposure
-    from lsst.daf.persistence import Butler
+    from lsst.daf.butler import Butler
 
-    # This is an example dataset on lsstdev which may be out of date,
-    # replace with a local dataset
-    DATA_DIR = "/datasets/hsc/repo/rerun/RC/w_2018_22/DM-14547"
-    butler = Butler(inputs=DATA_DIR)
+    DATA_DIR = "/repo/main"
```

Comment:
I don't think that `/repo/main` works anywhere ...

Reply:
It's meant to be an example. Also, K-T was pondering defining `/repo/main` as a butler alias at SLAC.

Reply:
Hmm. Nothing is going to work without a collections parameter either here or in the `fromButler` method.